jetty-ee10/jetty-ee10-servlet/src/main/java/org/eclipse/jetty/ee10/servlet/AsyncContentProducer.java

```diff
@@ -33,7 +33,7 @@
 class AsyncContentProducer implements ContentProducer
 {
     private static final Logger LOG = LoggerFactory.getLogger(AsyncContentProducer.class);
-    private static final Content.Chunk.Error RECYCLED_ERROR_CHUNK = Content.Chunk.from(new StaticException("ContentProducer has been recycled"));
+    private static final Content.Chunk RECYCLED_ERROR_CHUNK = Content.Chunk.from(new StaticException("ContentProducer has been recycled"));
```

Comment:
This chunk should be a `last` failure, shouldn't it?

Reply:
It is by default. Should I made it explicit?

Reply:
Probably not, but I got confused because `from(Throwable failure)` does not state that the last flag is set to true by default in the javadoc, while the implementation does the right thing.